js/start.js

```diff
@@ -1,6 +1,50 @@
 /** global: Clipboard */
 $(document).ready(function () {
-	new Clipboard('.copy-link');
+	var clipboard = new Clipboard('.copy-link');
+	clipboard.on('success', function(e) {
+		var $input = $(e.trigger);
+		$input.tooltip('hide')
+			.attr('data-original-title', t('core', 'Copied!'))
+			.tooltip('fixTitle')
+			.tooltip({placement: 'bottom', trigger: 'manual'})
+			.tooltip('show');
+		_.delay(function() {
+			$input.tooltip('hide');
+			if (OC.Share.Social.Collection.size() == 0) {
+				$input.attr('data-original-title', t('core', 'Copy'))
+					.tooltip('fixTitle');
+			} else {
+				$input.tooltip("destroy");
+			}
+		}, 3000);
+	});
+	clipboard.on('error', function (e) {
+		var $input = $(e.trigger);
+		var actionMsg = '';
+		if (/iPhone|iPad/i.test(navigator.userAgent)) {
+			actionMsg = t('core', 'Not supported!');
+		} else if (/Mac/i.test(navigator.userAgent)) {
+			actionMsg = t('core', 'Press ?-C to copy.');
```

Comment:
This should also be ⌘-C shouldn´t it? We should only have this cliboard code once and share it between vote.js and start.js.

Reply:
I don't know, and didn't think about this too much. I just copied and pasted from files app. Actually we should think about creating more helper classes, to avoid code doubling. But I would prefer to do this in the next step.

Reply:
Ok, i´m fine with that. But the question mark should be fixed before release. In vote.js it´s ⌘-C so it should be the same here.

Reply:
Oops. Done